### DISCUSSION TITLE
Should I create a separate project per camera for 3D pose estimation using multiple cameras in SLEAP and Anipose?

### DISCUSSION THREAD

**Discussion by gen21kawa** [talmolab/sleap#1591] [Help!] [answered]
Hi all! I would like to get some help on working with multiple cameras with different viewpoints. I’m working on a project on 3D pose estimation of a mouse doing a motor task. To this end, we are recording the behavior of a single mouse from 5 different synchronized cameras, which are located at 5 different positions around the mouse.

I created a project and imported videos from the 5 cameras, and I was initially annotating 10-20 frames from each video in the same project. But, when I was training the model, I noticed that the model was trained with frames from all the cameras. After the training, I got the message saying that no instance was found from the validation images because the model was trained with images from 5 different cameras. 

Because images captured from each camera is quite different, I thought that it would make sense to train a separate model for each camera, but I was wondering what the best practice would be. 

Should I create a different project for each camera? Or is there a way to import videos from all the 5 cameras in the same project but train a separate model for each camera?

**Comment by roomrys** [comment, marked as answer]
Hi @gen21kawa,

Currently, the models do not take into account different views for 3D. I think you will have better luck training individual models per each view since getting the model to generalize across views would take more annotations.

 And, if you want to try your luck at generalizing a single model, you could periodically combine all the projects from different views (Merge Into Project in menubar) to train one big model. There is NOT a similar reverse operation for splitting the projects up.
 
 Thanks,
 Liezl

**Reply by gen21kawa** [reply]
Hi Liezl! 

Thank you so much for the response! I will create a separate project for each camera first and might give it a shot for creating a single general model later.
